src/main/java/com/browserstack/pages/FavUserHomePage.java

```diff
@@ -0,0 +1,29 @@
+package com.browserstack.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class FavUserHomePage {
+
+    WebDriver driver;
+    public final static String OrderButtonLocator = "orders";
```

Comment:
CONSTANT

Reply:
All variables that are declared as final renamed to be in ALL CAPS